examples/2d/bounding_2d.rs

```diff
@@ -284,15 +284,19 @@ fn setup(mut commands: Commands, loader: Res<AssetServer>) {
     );
 }
 
+fn draw_filled_circle(gizmos: &mut Gizmos, position: Vec2, color: Srgba) {
+    for r in [1., 2., 3.] {
+        gizmos.circle_2d(position, r, color);
+    }
+}
```

Comment:
Previously, this logic appeared in two places. I pulled it into a function with a name that explains what it's trying to do.